includes/helper-functions.php

```diff
@@ -990,3 +990,18 @@ function edac_generate_summary_stat( string $item_class, int $count, string $lab
             <div class="edac-panel-number-label">' . $label . '</div>
         </li>';
 }
+
+/**
+ * Check if an element has an extension that matches the provided list.
+ *
+ * @since 1.15.0
+ *
+ * @param string $item A file path or URL to check.
+ * @param array  $extensions An array of extensions to check for.
+ *
+ * @return bool
+ */
+function is_item_using_matching_extension( string $item, array $extensions ): bool {
+	$extension = pathinfo( $item, PATHINFO_EXTENSION );
+	return in_array( '.' . $extension, $extensions, true );
```

Comment:
Are extensions always passed with the dot included?

Maybe we should always strip the dot from the extension to ensure passing with or without the dot works?

Reply:
With pathinfo() and PATHINFO_EXTENSION, the extension is always omitted. I also wondered if we could strip the dot from the $extensions arrays, but I figured it could have some unintended consequences that would require more thorough testing.